src/LinuxTracing/UprobesUnwindingVisitor.h

```diff
@@ -126,6 +132,10 @@ class UprobesUnwindingVisitor : public PerfEventVisitor {
                    orbit_grpc_protos::Callstack* resulting_callstack,
                    bool offline_memory_only = false);
 
+  template <typename CallchainPerfEventDataT>
+  bool VisitCallchainEventInternally(const CallchainPerfEventDataT& event_data,
```

Comment:
I would prefer "Internal", or just `VisitCallchainEvent`, similarly to `SwitchesStatesNamesVisitor::VisitSchedSwitch` and `SwitchesStatesNamesVisitor::VisitSchedWakeup`.